doc/sphinx/source/releasenotes.md

```diff
@@ -31,13 +31,15 @@ for each release of libCEED.
 - `CeedDebugEnv()` macro created to provide debugging outputs when Ceed context is not present.
 - Added {c:func}`CeedStringAllocCopy` to reduce repeated code for copying strings internally.
 - Added {c:func}`CeedPathConcatenate` to facilitate loading kernel source files with a path relative to the current file.
+- Added {c:func}`CeedBasisCreateHdiv` convenience constructor and CPU backend support for non-tensor H(div) elements.
```

Comment:
We actually added support for H(div), which is a major feature. The `CeedElemTopology` entries were all renamed.

Reply:
updated